sentry-errors.md

```diff
@@ -0,0 +1,19 @@
+Process for triaging new errors on Sentry
+=========================================
+
+Make it part of your process to go to Sentry and go to the "Needs Triage" view of the production project.
+These are all the issues that nobody looked at yet. Check them out and do the following.
+
+* It is an error that can be fixed:
+  * Create a GH issue and link the Sentry error to it.
```

Comment:
is it possible to add mantecao issues as well - or is the resmio repo the only one linked? Maybe we should follow the issue in the resmio repo as well, but I guess worth thinking about. It would probably be best to create an issue in the resmio repo for updating our mantecao package - then we can fix the problem with a new mantecao release..?